src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/StatusService.scala

```diff
@@ -36,22 +37,34 @@ class StatusService(gdDAO: DataprocDAO,
       GoogleDataproc -> checkGoogleDataproc,
       Sam -> checkSam,
       Database -> checkDatabase
-    )
+    ).map(logFailures.tupled)
   }
 
+  // Logs warnings if a subsystem status check fails
+  def logFailures: (Subsystem, Future[SubsystemStatus]) => (Subsystem, Future[SubsystemStatus]) = (subsystem, statusFuture) =>
+    subsystem -> statusFuture.andThen {
+      case Success(status) if !status.ok =>
+        logger.warn(s"Subsystem [$subsystem] reported error status: $status")
```

Comment:
I'd be tempted to push all of these in this file down one level -- i.e. warn -> debug, debug -> info.

Reply:
I wanted to warn on errors so they show up in logs, and debug OK statuses (the more common case) so they don't fill up logs in production.